src/main/java/com/extendedclip/deluxemenus/config/DeluxeMenusConfig.java

```diff
@@ -57,11 +59,12 @@ public class DeluxeMenusConfig {
     VALID_MATERIALS.addAll(PLAYER_ITEMS);
     VALID_MATERIALS.add(WATER_BOTTLE);
 
-    VALID_MATERIAL_PREFIXES.addAll(HEAD_PREFIXES);
-    VALID_MATERIAL_PREFIXES.add(PLACEHOLDER_PREFIX);
-    VALID_MATERIAL_PREFIXES.add(ITEMSADDER_PREFIX);
-    VALID_MATERIAL_PREFIXES.add(ORAXEN_PREFIX);
-    VALID_MATERIAL_PREFIXES.add(MMOITEMS_PREFIX);
+    VALID_MATERIAL_PREFIXES.addAll(
+        DeluxeMenus.getInstance().getItemHooks().values()
+            .stream()
+            .map(ItemHook::getPrefix)
+            .collect(Collectors.toList())
+    );
```

Comment:
does this work? surely DeluxeMenus hasnt initialized before the static block is ran and definitely not the item hooks?

Reply:
I had to shuffle around when the plugin setups item hooks to be initialized before the config is to allow this to work.

![image](https://github.com/HelpChat/DeluxeMenus/assets/38044508/c568a9b4-e134-44f4-a814-7d9c86342530)
![image](https://github.com/HelpChat/DeluxeMenus/assets/38044508/dc7e388c-91ec-445e-aa37-8b0a31da1275)